### DISCUSSION TITLE
Multi-Handle Range Sliders

### DISCUSSION THREAD

**Discussion by Niamh1** [skeletonlabs/skeleton#2488] [Q&A] [answered]
Is it possible to use the current RangeSlider to bind two handles, instead of one? 
<img width="298" alt="range_slider" src="https://github.com/skeletonlabs/skeleton/assets/3873524/a04c8b7f-9887-4712-b4ef-2ccbd0c2278a">

**Comment by Sarenor** [comment, marked as answer]
The RangeSlider adheres very closely to the native input right now:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range
The native input can only be bound to one handle though.

It is under consideration for Skeleton v3 though: https://github.com/skeletonlabs/skeleton/issues/2367
